pennylane_lightning/src/tests/Makefile

```diff
@@ -24,13 +24,16 @@ all: test
 lightning_unittest.o: lightning_unittest.cpp
 	$(CC) $^ $(CFLAGS) -c
 
-lightning_util_unittest.o: lightning_util_unittest.cpp
```

Comment:
I think we still want to keep `lightning_util_unittest`?

Reply:
Great catch! Changed